item-test/test_refda_stdio.py

```diff
@@ -361,4 +361,51 @@ def test_odm(self):
         # items of the report
         self.assertEqual(1, len(rpt.items))
         self.assertEqual(6, rpt.items[0].value.size);
-        self.assertEqual(False, rpt.items[0].value[0][5].value); # Confirm rule is disabled
\ No newline at end of file
+        self.assertEqual(False, rpt.items[0].value[0][5].value); # Confirm rule is disabled
+
+
+    def test_odm_var_const(self):
+        self._start()
+
+        LOGGER.setLevel(logging.INFO)
+
```

Comment:
It's useful in some testing to assert preconditions, in this case inspect the `var-list` and see that it is empty to start with. That ensures that what is tested is that `ensure-var` actually adds something.

Reply:
Added preconditions for VAR and CONST.